### PR TITLE
Fix CMakeLists.txt in libs/sockets to build on buster

### PR DESCRIPTION
**Pull request by iliaplatone**

### CODE REVIEW

**Comment by pawel-soja** [comment]
Should I add Debian Buster to CI?

**Comment by iliaplatone** [comment]
I'm building on that, if you want yes.

**Comment by pawel-soja** [comment]
For the sake of readability of `CMakeLists.txt`,
I would very much like to skip the constant repetition of `${CMAKE_CURRENT_SOURCE_DIR}`.

For Debian Buster, `cmake` is version `3.13` and `${CMAKE_CURRENT_SOURCE_DIR}` can be omitted.
In the current pull request https://github.com/indilib/indi/pull/1767 I'm working on, I introduced **CI for the Debian Buster** system and made additional **fixes** to make `indiclientqt` compile for the older version of `Qt`

@iliaplatone  Could you please check if everything is ok?

**Comment by iliaplatone** [comment]
/home/plato/dev/github/indi/libs/indibase/baseclient_p.h:6:10: fatal error: tcpsocket.h: No such file or directory
 #include <tcpsocket.h>

File: libs/sockets/CMakeLists.txt [hunk 1]
```diff
@@ -38,7 +38,7 @@ target_sources(${PROJECT_NAME}
 )
 
 target_include_directories(${PROJECT_NAME}
-    PUBLIC ${CMAKE_CURRENT_SOURCE_DIR}/
+    PUBLIC
```
Comment (pawel-soja) [inline libs/sockets/CMakeLists.txt]: missing dot
```
PUBLIC .
```

**Comment by iliaplatone** [comment]
It needs that for the headers at least

**Comment by iliaplatone** [comment]
Oh.. some mess over there forgot to point to ${CMAKE_CURRENT_SOURCE_DIR}/ the include path

**Comment by pawel-soja** [comment]
Is the version change not enough?
I'm testing it on Debian Buster (docker):
```cmake
cmake_minimum_required(VERSION 3.13)
project(sockets CXX)

find_package(Threads REQUIRED)
add_library(${PROJECT_NAME} STATIC "")

# Headers
list(APPEND ${PROJECT_NAME}_HEADERS
    tcpsocket.h
)

list(APPEND ${PROJECT_NAME}_PRIVATE_HEADERS
    tcpsocket_p.h
)

# Sources
list(APPEND ${PROJECT_NAME}_SOURCES
    tcpsocket.cpp
)

if(WIN32)
    list(APPEND ${PROJECT_NAME}_SOURCES
        tcpsocket_win.cpp
    )
else()
    list(APPEND ${PROJECT_NAME}_SOURCES
        tcpsocket_unix.cpp
    )
endif()

# Setup Target
target_sources(${PROJECT_NAME}
    PUBLIC
    ${${PROJECT_NAME}_SOURCES}
    ${${PROJECT_NAME}_HEADERS}
    PRIVATE
    ${${PROJECT_NAME}_PRIVATE_HEADERS}
)

target_include_directories(${PROJECT_NAME}
    PUBLIC ./
)

target_link_libraries(${PROJECT_NAME} Threads::Threads)
```

File: libs/sockets/CMakeLists.txt [hunk 2]
```diff
@@ -38,7 +38,7 @@ target_sources(${PROJECT_NAME}
 )
 
 target_include_directories(${PROJECT_NAME}
-    PUBLIC 
+    PUBLIC ${CMAKE_CURRENT_SOURCE_DIR}/
```
Comment (pawel-soja) [inline libs/sockets/CMakeLists.txt:41]: Please check the original version:
```cmake
    PUBLIC ./
```

File: libs/sockets/CMakeLists.txt (hunk 1 above)
Comment (iliaplatone) [reply libs/sockets/CMakeLists.txt]: Just a bit nostalgic

**Comment by iliaplatone** [comment]
Done, it builds here also.

**Comment by pawel-soja** [comment]
To confirm, it was enough to change the version only?

Do you want to merge this pull request as soon as possible?
Can you wait for the upcoming big changes (include `Debian Buster`) in https://github.com/indilib/indi/tree/refactor/cmake-drivers

**Comment by iliaplatone** [comment]
Yes that was enough. I can wait.

**Pull request closed without merging**